packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -238,18 +239,20 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
   }
 
   return (
-    <View {...containerProps}>
-      <View {...iconAndMessageContainer}>
-        <View style={iconViewStyle}>
-          <Icon {...iconProps} />
+    <ComponentWrapper>
```

Comment:
Just adding ComponentWrapper to ensure component always is pulling i18n similar to other components. Worked fine in-app because they already had the requisite translations.